CHANGELOG.md

```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#10293](https://github.com/MetaMask/metamask-extension/pull/10293): Implement Ledger Live bridge
+- [#10962](https://github.com/MetaMask/metamask-extension/pull/10962): Add backwards compatibility for ETH <-> WETH contract address validation
 
-## [9.5.1]
-### Uncategorized
-- [#11056](https://github.com/MetaMask/metamask-extension/pull/11056): Sync `master` with `develop`
-- Fixing changelog, bumping package.json -> 9.5.1
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- [#11051](https://github.com/MetaMask/metamask-extension/pull/11051): Fix Sentry sourcemaps
-- [#11049](https://github.com/MetaMask/metamask-extension/pull/11049): Use isValidAddress helper instead of ethereumJsUtil.isValidAddress, when appropriate
-- [#11034](https://github.com/MetaMask/metamask-extension/pull/11034): Bump hosted-git-info from 2.5.0 to 2.8.9
-- [#11016](https://github.com/MetaMask/metamask-extension/pull/11016): Group signature transactions
-- [#11009](https://github.com/MetaMask/metamask-extension/pull/11009): Replace uses of `hasOwnProperty.call` with simpler alternatives
-- [#11031](https://github.com/MetaMask/metamask-extension/pull/11031): Fix error behavior of addEthereumChain
-- [#11030](https://github.com/MetaMask/metamask-extension/pull/11030): Update switchEthereumChain unrecognized chain error
-- [#11029](https://github.com/MetaMask/metamask-extension/pull/11029): Migrate version from `_base` manifest to `package.json`
-- [#10905](https://github.com/MetaMask/metamask-extension/pull/10905): rough in switchEthereumChain
-- [#11025](https://github.com/MetaMask/metamask-extension/pull/11025): Fix redirection to the build quotes page from the swaps failed page
-- [#11011](https://github.com/MetaMask/metamask-extension/pull/11011): Update jsconfig excludes array
+### Changed
+- [#11007](https://github.com/MetaMask/metamask-extension/pull/11007): Hide basic tab in advanced gas modal when on testnets.
+- [#10994](https://github.com/MetaMask/metamask-extension/pull/10994): Replacing the text "Seed Phrase" with "Secret Recovery Phrase" throughout the application.
+- [#10987](https://github.com/MetaMask/metamask-extension/pull/10987): Swaps: Improve hardware wallet UX
+- [#10936](https://github.com/MetaMask/metamask-extension/pull/10936): Swaps: Allow 0% slippage, show a warning for 0 < slippage <= 1, disallows Reviews when negative slippage occurs.
+- [#10946](https://github.com/MetaMask/metamask-extension/pull/10946): Send user to activity view after transaction is complete
+- [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the "Need Help" text to the bottom of the full page view
+- [#10767](https://github.com/MetaMask/metamask-extension/pull/10767): Improving error handling when gas prices are unavailable.
+- [#10836](https://github.com/MetaMask/metamask-extension/pull/10836): Improving "Review Swap" error handling
```

Comment:
`[#10836](https://github.com/MetaMask/metamask-extension/pull/10836): Improving "Review Swap" error handling` should not be included in this changelog (was part of v9.5.0)

Reply:
fixed